src/Core/Models/Api/Request/AttachmentRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+namespace Bit.Core.Models.Api
+{
+    public class AttachmentRequestModel
+    {
+        public string key { get; set; }
```

Comment:
(K/k)ey?

Reply:
not sure if you're commenting on the capitalization or the presence of a key.

It should be capitalized, thank you.
As for presence, this is how attachments work to enable sharing to organization at a later date. Similar to Sends, each attachment is encrypted with a key unique to that attachment. The key is stored encrypted as a part of a cipher's data. That way, when sharing a cipher, we don't need to delete/reupload an attachment associated with that cipher, we just share the key and encrypt it as a part of the org rather than the personal Cipher item.

Reply:
I was just commenting on the capitalization, but the explanation is appreciated too 😄